plugins/field-slider/test/field_test_helpers.js

```diff
@@ -0,0 +1,138 @@
+/**
+ * @license
+ * Copyright 2020 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+const assert = require('chai').assert;
+const Blockly = require('blockly');
+
+
+/**
+ * @typedef {{
```

Comment:
Nice, add comment for what this is a typedef for.

Reply:
Done.